### ISSUE TITLE
Wrongly formated contact makes application crash

### ISSUE THREAD

**Issue by clochix** [cozy/cozy-emails#686]
A contact in user database has no datapoints, which make the client crash:
```
EROR +00259 controllers:activity { type: 'error',
  error: 
   { msg: 'rawResult.datapoints is undefined',
     name: 'TypeError',
     full: 'TypeError: rawResult.datapoints is undefined',
     stack: 'ContactStore</_import/convert/<@http://localhost:9125/js/app.js:12498:11\nContactStore</_import/convert@http://localhost:9125/js/app.js:12495:1\nt/<.withMutations@http://localhost:9125/js/vendor.js:55956:458\nContactStore</_import@http://localhost:9125/js/app.js:12518:18\nContactStore<@http://localhost:9125/js/app.js:12523:3\n@http://localhost:9125/js/app.js:12467:17\ninitModule@http://localhost:9125/js/vendor.js:64:1\nrequire@http://localhost:9125/js/vendor.js:74:1\nlocalRequire/<@http://localhost:9125/js/vendor.js:57:14\n@http://localhost:9125/js/app.js:12915:16\ninitModule@http://localhost:9125/js/vendor.js:64:1\nrequire@http://localhost:9125/js/vendor.js:74:1\nlocalRequire/<@http://localhost:9125/js/vendor.js:57:14\n@http://localhost:9125/js/app.js:13954:16\ninitModule@http://localhost:9125/js/vendor.js:64:1\nrequire@http://localhost:9125/js/vendor.js:74:1\nlocalRequire/<@http://localhost:9125/js/vendor.js:57:14\nwindow.onload@http://localhost:9125/js/app.js:10112:24\n' },
  file: 'http://localhost:9125/js/app.js',
  line: 12498,
  col: 11,
  href: 'http://localhost:9125/' }

```

We should handle the case where contact has no datapoints. Also we shouldn't crash if we are unable to import some contacts.

**Comment by kimmy1** [comment]
So what does all this mean is he cheating or not

**Comment by poupotte** [comment]
This issue is fixed in new version (0.9.18)
